firestore/src/City.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/**
+ * Copyright 2022 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * For instructions on how to run the full sample:
+ *
+ * @see https://github.com/GoogleCloudPlatform/php-docs-samples/tree/master/firestore/README.md
+ */
+
+namespace Google\Cloud\Samples\Firestore;
+
+# [START firestore_data_custom_type_definition]
+class City
+{
+    public $name;
+    public $state;
+    public $country;
+    public $capital;
+    public $population;
+    public $regions;
+
+    public function __construct(
+        $name,
+        $state,
+        $country,
+        $capital = false,
+        $population = 0,
+        $regions = []
```

Comment:
we should be using typehints for everything. Right now it's really hard to know what the types are, and which ones can be nullable. 
```suggestion
        string $name,
        string $state,
        string $country,
        bool $capital = false,
        int $population = 0,
        array $regions = []
```